lang/sv/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: swedish
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
```

Comment:
You're free to propose a change, I don't have the time to do it myself, just gave it a quick read when I saw a tweet from @haacked